core/src/main/java/dev/keva/core/command/impl/string/GetEx.java

```diff
@@ -0,0 +1,111 @@
+package dev.keva.core.command.impl.string;
+
+import dev.keva.core.command.annotation.CommandImpl;
+import dev.keva.core.command.annotation.Execute;
+import dev.keva.core.command.annotation.ParamLength;
+import dev.keva.core.exception.CommandException;
+import dev.keva.ioc.annotation.Autowired;
+import dev.keva.ioc.annotation.Component;
+import dev.keva.protocol.resp.reply.BulkReply;
+import dev.keva.protocol.resp.reply.Reply;
+import dev.keva.store.KevaDatabase;
+
+import java.nio.charset.StandardCharsets;
+import java.sql.Timestamp;
+
+@Component
+@CommandImpl("getex")
+@ParamLength(type = ParamLength.Type.AT_LEAST, value = 2)
+public class GetEx {
+    private final KevaDatabase database;
+
```

Comment:
consider put these flags in some common ENUM variables

Reply:
yeah should do so